css-two-column-layout/styles.css

```diff
@@ -0,0 +1,108 @@
+
+.container{
+
+width: 700px;
+margin: auto;
+background-color: white;
+
+
+}
+.row {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+
+
+
```

Comment:
Please remove empty lines in your CSS rules, with the exception of one empty line between different rules.